docs/code conventions (#905)
scania-digital-design-system/sdds: .github/CODE_STYLE.md

 - Use meaningful variable and function names: Name your variables and functions in a way that clearly communicates their purpose.
Avoid using single-letter variable names, instead of `i` for index, be descriptive and name the variable `index`.
 - Use single quotes (') for strings.

## Folder structure

All components are located in the `src/components` directory. Each component in this folder has a dedicated folder named as the
componenet minus the `sdds`-prefix. For our button the file path is `src/components/button`. This folder should have two subdirectories,
one for the web component and one for the HTML/CSS version of the component. For the `<sdds-button>` it is
`src/components/button/web-component` and for the HTML/CSS version of this component it is `src/components/button/css-component`. Since
these components share some css variables, these are located in a scss file in the `src/components/button` directory. The name of the variables
file is 'component-name'-vars.scss (`button-vars.scss`).

### File naming

For all components that are pure HTML/CSS components their files should be prefixed with `css-`. This is to clearly separeate what files are part

> of our web components and which are only used for HTML/CSS components. For our button the means that the styling related to the HTML/CSS version 

Reviewer: I'm a bit confused with this sentence "For our button the means", should it be "For our button that means" instead?

> of our web components and which are only used for HTML/CSS components. For our button the means that the styling related to the HTML/CSS version 
> of our component the file name is `css-button.scss`. 

Reviewer 2: ```suggestion
of our web components and which are only used for HTML/CSS components. For our button, when it comes to styling related to the HTML/CSS version 
of our component, the file name is `css-button.scss`. 
```
